src/track/heaptrack.sh.cmake

```diff
@@ -100,11 +106,21 @@ while true; do
                 echo "Missing output argument."
                 exit 1
             fi
-            output=$(readlink -f $2)
+            output=$2
+            if [ -n "$(echo $output | grep '%h')" ]; then
+              output=$(echo $output | sed "s/%h/$(hostname)/g")
+            fi
+            if [ -n "$(echo $output | grep '%g')" ]; then
```

Comment:
`%g` isn't documented, should this be `%p` instead?

Reply:
Yes, I should have written %p.